docs/UserGuide.md

```diff
@@ -77,27 +77,31 @@ Format: `help`
 
 ### Adding a person: `add`
 
-Adds a person to the address book.
+Adds a person to the contact list in FINDvisor.
 
 Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-A person can have any number of tags (including 0)
+A person can have any number of tags (including 0) 
+pertaining to how many financial plans he/she is under
```

Comment:
Should we explicitly state that it is for financial plans? Since tags can be used for general purpose.
Also a nit but good to add punctuation at the end of sentence for consistency.